systems/Query/07-Deployment_View.md

```diff
@@ -0,0 +1,53 @@
+# Deployment View
+
+## Wikidata.org
+
+[Wikidata.org](./../../Glossary.md#wikidata) is the largest deployment of the Query Service.
+It makes use of the Streaming Updater, and runs on multiple servers, in multiple distinct groups, in multiple data centres hosted by the Wikimedia Foundation.
+
+Canonical documentation:
+
+- [Wikidata Query Service deployment page](https://wikitech.wikimedia.org/wiki/Wikidata_Query_Service)
+- [Wikimedia Event Platform](https://wikitech.wikimedia.org/wiki/Event_Platform)
+
+### Multi Cluster
+
+![](./diagrams/07-wikidata-deployment.drawio.svg)
+
+### Single Host
+
+Each server in this setup runs the Streaming Updater Consumer, getting a stream of RDF changes from Kafka, that have been created by a single Streaming Updater Producer.
+
+![](./diagrams/07-wikidata-deployment-single-host.drawio.svg)
+
+### User Interfaces
+
+User Interfaces are deployed to [Microsites](https://wikitech.wikimedia.org/wiki/Microsites) infrastructure.
+
+[Wikitech documentation exists for this deployment](https://wikitech.wikimedia.org/wiki/Microsites#Wikidata_Query_Service_(UIs)).
+
+The application is built by Jenkins and committed to a new Git repository, which is then deployed ot Microsites.
+
+![](./diagrams/07-wikidata-deployment-user-interfaces.drawio.svg)
+
+## Wikibase Docker
+
+Wikibase docker releases, and the suggested 3rd party deployments make use of the older Recent Changes based updater.
+
+Containers can be deployed in any number of ways.
+
+The docker-compose example would result in these containers all being within the same host machine.
+
+![](./diagrams/07-wikibase-docker.drawio.svg)
+
+## Wikibase as a Service
```

Comment:
While I agree that both sections for wikidata.org and for Wikibase Docker make sense in this Deployment view, I am not convinced that the WBaaS architecture has a reasonable purpose here. In any case the entire diagram is way out-of-scope for the QS documentation and it can't even explain what "specific needs" the `query-gateway` and the `query-updater` fulfill or what exactly they do.